process_request/helpers.py

```diff
@@ -122,9 +122,10 @@ def get_preferred_format(item_json):
     preferred = None, None, None, None
     if item_json.get("instances"):
         instances = item_json.get("instances")
-        if "digital_object" in instances:
+        print(instances)
```

Comment:
Did we want to leave this in here for now?

Reply:
Nope, that's a mistake. Thanks for the catch!